packages/bot-web-ui/src/components/self-exclusion.jsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+    Input,
+    ThemedScrollbars,
+    Button,
+    Modal,
+    MobileWrapper,
+    Div100vhContainer,
+    FadeWrapper,
+    PageOverlay,
+} from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { Formik, Form, Field } from 'formik';
+import { connect } from '../stores/connect';
+
+class SelfExclustionForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            errors: '',
+        };
+    }
+    render() {
+        const {
+            initial_values,
+            max_losses,
+            onRunButtonClick,
+            run_limit,
+            set_max_losses,
+            setLimitations,
+            ToggleSelfExclusion,
+            updateSelfExclusion,
+            is_mobile,
+        } = this.props;
+        const form_errors = this.state.errors;
+        const setErrors = () => {
+            const errors = {};
+            if (!max_losses && !set_max_losses) {
+                errors.set_max_losses = localize('This field is required.');
+            }
+            if (!run_limit) {
+                errors.run_limit = localize('This field is required.');
+            }
+            this.setState({ errors });
+        };
+
+        const onSubmitLimits = async () => {
+            this.setState({ errors: {} });
+            setErrors();
+            if (set_max_losses && set_max_losses !== max_losses) {
+                const set_losses = await updateSelfExclusion({ max_losses: set_max_losses });
+                if (set_losses?.error) {
+                    this.setState(pre => ({
+                        errors: {
+                            ...pre.errors,
+                            set_max_losses: localize(set_losses.error.message),
+                        },
+                    }));
+                    return;
+                }
+                setLimitations('max_losses', set_max_losses);
+            }
+            onRunButtonClick();
+        };
+
+        const onFormChange = (type, e) => {
+            setLimitations(type, e.currentTarget.value);
+        };
+        return (
+            <div className='self-exclusion'>
+                <div className='self-exclusion__content'>
+                    <ThemedScrollbars
+                        autohide
+                        style={{
+                            height: is_mobile ? 'calc(100vh - 250px)' : '246',
```

Comment:
Please checkout AutoHeightWrapper from the components package to get the height from the browser and not hard code it.

Reply:
@aarimm , @reza-fs  because I used fixed size for elements on desktop and also the components works fine on size I think it's not necessary to have themedscrollbars at all therefore I removed it